docs/sources/operators-guide/configure/configure-tsdb-block-upload.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Configuring TSDB block upload"
+menuTitle: "Configuring TSDB block upload"
+description: "Learn how to configure Grafana Mimir to enable TSDB block upload"
+weight: 120
+---
+
+# Configure TSDB block upload
+
+Grafana Mimir supports uploading of historic TSDB blocks, sourced from f.ex. Prometheus or
+Grafana Mimir itself.
+
+**Note**: Please read about limitations related to importing blocks from Thanos in
+[Migrating from Thanos or Prometheus to Grafana Mimir]({{< relref "../../migration-guide/migrating-from-thanos-or-prometheus.md" >}}).
+
+The functionality is disabled by default, but can be enabled via the `-compactor.block-upload-enabled`
+CLI flag, or via the corresponding `limits.compactor_block_upload_enabled` configuration parameter:
+
+```yaml
+limits:
+  # Enable TSDB block upload
+  compactor_block_upload_enabled: true
+```
+
+## Enabling TSDB block upload per tenant
+
+If your Grafana Mimir has multitenancy enabled, you can still use the preceding method to enable
+TSDB block upload for all tenants. If instead you wish to enable it per tenant, you can use the
+runtime configuration to set a per-tenant override:
+
+1. Enable [runtime configuration]({{< relref "about-runtime-configuration.md" >}}).
+1. Add an override for the tenant that should have TSDB block upload enabled:
```

Comment:
I think this edit is misleading, because the tenant doesn't have block upload enabled *until* the override is added. "Should" conveys the correct meaning here.